README.md

```diff
@@ -29,16 +29,28 @@ Table of Contents
 # AutoSpotting #
 
 A simple tool designed to significantly lower your Amazon AWS costs by
-automating the use of the [spot market](https://aws.amazon.com/ec2/spot). It can
-often achieve savings up to 90% off the usual on-demand prices, like shown in
-the screenshot below.
+automating the use of the [spot market](https://aws.amazon.com/ec2/spot).
 
-When enabled on your existing on-demand AutoScaling group, it starts launching
+When enabled on an existing on-demand AutoScaling group, it starts launching
 EC2 spot instances that are cheaper, at least as powerful and configured as
 closely as possible as your existing on-demand instances.
 
-It then gradually swaps them with your existing on-demand instances, which can
-then be terminated.
+It then gradually swaps them one by one with your existing on-demand instances.
+
+![Workflow](https://cdn.cloudprowess.com/images/autospotting.gif)
+
+In the above example:
+- it was enabled on a group of three instances.
+- configured to keep an on-demand instance running at all times.
+- each of the two spot instances were roughly 2x cheaper, had 4x more CPU
+  cores and 2x more RAM that the on-demand instances they replaced.
+- overall we got 3x more CPU cores and 66% more RAM than we had initially, and
+  all this with 33% cost savings.
+
+In all-spot configurations, it can often achieve savings up to 90% off
+the usual on-demand prices, like shown in the screenshot below:
```

Comment:
Yup, I was referring for the 'all-spot' configuration. Might be worth mentioning it that it was m3.medium replaced, by the same type in case other people wondered as well